pkg/migrate/build-manifests.go

```diff
@@ -0,0 +1,267 @@
+package migrate
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"strings"
+
+	publicv1 "github.com/cockroachdb/cockroach-operator/apis/v1alpha1"
+	"github.com/cockroachdb/errors"
+	"github.com/cockroachdb/helm-charts/pkg/upstream/cockroach-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type Manifest struct {
+	cloudProvider  string
+	cloudRegion    string
+	object         string
```

Comment:
How about calling this `objectName`, and renaming `objectManifest` to something like `objectPath` or `objectManifestPath`?